### ISSUE TITLE
Getting Started sample failing on Node 14.8

### ISSUE THREAD

**Issue by christopherdiehl** [blocknative/sdk#195]
Hello!

Trying to setup the sample on Node 14.8 locally provides the following error:

```
const blocknative = new BlocknativeSdk(options)
                    ^

TypeError: BlocknativeSdk is not a constructor
```

Complete code snippet:
```
import WebSocket from 'ws'
import BlocknativeSdk from 'bnc-sdk'
import Web3 from 'web3'

// const web3 = new Web3('<ws://some.local-or-remote.node:8546>')

// create options object
const options = {
  dappId: '<API>',
  networkId: 4,
  ws: WebSocket,
  // un-comment if you would like to log all transaction events
  transactionHandlers: [event => console.log(event.transaction)]
}

// initialize and connect to the api
const blocknative = new BlocknativeSdk(options)
```

Running via `node index.js`

Local information:
- node version == v14.18.0
- package.json type == module

**Comment by aaronbarnardsound** [comment]
Working on a fix for this currently!

**Comment by aaronbarnardsound** [comment]
Fix released in version [4.4.1](https://github.com/blocknative/sdk/releases/tag/4.4.1)
